mock-echo-app/src/cmr/mock_echo/client/echo_util.clj

```diff
@@ -79,7 +114,7 @@
   ([provider-guid coll-identifier]
    (catalog-item-id provider-guid coll-identifier nil))
   ([provider-guid coll-identifier gran-identifier]
-   {:provider-guid provider-guid
+   {:provider_id provider-guid
```

Comment:
The ACL schema requires _ delimited names.  Seems easier to do it this way than to use a string, right?